src/app/loginComponents/accounts/external/revoke-token-dialog/revoke-token-dialog.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, OnDestroy, Inject } from '@angular/core';
+import { AbstractControl, FormBuilder, FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { UserQuery } from '../../../../shared/user/user.query';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-delete-account-dialog',
+  templateUrl: './revoke-token-dialog.component.html',
+  styleUrls: ['./revoke-token-dialog.component.scss'],
+})
+export class RevokeTokenDialogComponent implements OnDestroy {
```

Comment:
If you extend `Base`, then you don't need to reimplement `OnDestroy`, `ngUnsubscribe`, etc.